Authorization/Authorization.xcodeproj/project.pbxproj

```diff
@@ -306,7 +295,6 @@
 				737436DC910C5A48E4C01AC7 /* [CP] Check Pods Manifest.lock */,
 				07169450296D913300E3DED6 /* Sources */,
 				07169451296D913300E3DED6 /* Frameworks */,
-				07169452296D913300E3DED6 /* Resources */,
```

Comment:
I'm unsure what's being removed here by the https://github.com/kronenthaler/mod-pbxproj package. It's automatic.